Expand `Beatmap/Beatmap collaborations` (#8952)
ppy/osu-wiki: wiki/Beatmap/Beatmap_collaborations/en.md

---
tags:
  - collab
  - collabs
---

# Beatmap collaborations

*For help starting a collaboration, see: [How to start a beatmap collaboration](/wiki/Guides/Collab_Information)*

**Beatmap collaborations**, or ***collabs***, are single [difficulties](/wiki/Beatmap/Difficulty) or [beatmaps](/wiki/Beatmap) with [guest difficulties](/wiki/Beatmap/Guest_difficulty) created by multiple people. While the term *collab* usually refers to beatmaps, it also means any other media created by several participants, such as [storyboards](/wiki/Storyboard) or songs.

## Description

> Beatmap collaborations usually have the "Collab" word or possession of multiple usernames in their difficulty name or [beatmap description](/wiki/Beatmap/Beatmap_description), although it is not a strict [ranking criteria](/wiki/Ranking_Criteria) requirement. It is also common for frequent participants to invent amalgamations of their usernames[^name-example] and use them consistently throughout the mapping career.

Reviewer: ```suggestion
Beatmap collaborations usually have the word "Collab" or multiple usernames attached to their difficulty name or [beatmap description](/wiki/Beatmap/Beatmap_description), although it is not a strict [ranking criteria](/wiki/Ranking_Criteria) requirement. It is also common for frequent participants to invent amalgamations of their usernames[^name-example] and use them consistently throughout their mapping career.
```

> The mapping usually starts with the [beatmap host](/wiki/Beatmap/Beatmap_host) splitting the pre-[timed](/wiki/Beatmapping/Timing) beatmap template into several parts[^collab-split] and assigning the said parts among the collaborators according to a shared agreement. Mappers complete[^blind-collab] their parts by laying out the [hit objects](/wiki/Gameplay/Hit_object), applying [hitsounds](/wiki/Beatmapping/Hitsound), using [inherited timing points](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) where necessary, and then pass their work back to the host, who then combines the changes into a single difficulty by carefully editing the difficulty's [`.osu` file](/wiki/Client/File_formats/Osu_(file_format)).

Reviewer: ```suggestion
The mapping usually starts with the [beatmap host](/wiki/Beatmap/Beatmap_host) splitting a pre-[timed](/wiki/Beatmapping/Timing) beatmap template into several parts[^collab-split] and assigning them to the collaborators according to some agreed-upon distribution. Mappers complete[^blind-collab] their parts by laying out the [hit objects](/wiki/Gameplay/Hit_object), applying [hitsounds](/wiki/Beatmapping/Hitsound), using [inherited timing points](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) where necessary, and then pass their work back to the host, who then combines the changes into a single difficulty by carefully editing its [`.osu` file](/wiki/Client/File_formats/Osu_(file_format)).
```

## Notes and references

[^name-example]: ["masazi — mayday B", a beatmap by Vass_Bass](https://osu.ppy.sh/beatmapsets/74277#osu/233068)
[^collab-split]: The song is marked up using [timeline](/wiki/Client/Beatmap_editor/Timelines) bookmarks according to its musical structure. Relatively long segments, such as verses, are usually split after every 4 or 8 [measures](/wiki/Music_theory/Measure).

> [^blind-collab]: Although mappers usually rely on the parts already completed to make gameplay transitions as smooth as possible, there exist so-called blind collabs such as [Mitose Noriko - Unreal Player](https://osu.ppy.sh/beatmapsets/828563#osu/1735958) hosted by ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515).

Author: dunno they read as side comments to me. maybe `collab-split` may be properly included into the body but I don't know how
